typings/dtos.ts

```diff
@@ -1231,6 +1231,18 @@ export interface MatchV4ParticipantStatsDto {
     totalHeal?: number // int64
     totalMinionsKilled?: number // int32
     timeCCingOthers?: number // int64
+    /**
+     * Offense rune stat bonus
+     */
+    statPerk0: number // int32
```

Comment:
For this, try running `yarn swagger` to automatically generate the types and let me know if they are the same.

Reply:
Never mind, I'm guessing it doesn't (you already caught it):

> I also added the rune bonuses to the MatchV4ParticipantStatsDto. These are not on the official documentation page yet, but they are returned in the data.

Reply:
Oh, I didn't realize this was auto-generated. I have not used many of the tools here before.

I ran `yarn swagger` and the statPerk fields disappear as expected. The V3 interfaces also get removed as well as some more minor changes to other interfaces.

I took a brief look at the riotapi-schema project which it pulls from. Would it be worth it to look into using a manual DTO override?

Reply:
Sorry, I missed this. I'm fine with manual DTO overrides! I want TypeScript support to be better.